source/locale/en/symbols.dic

```diff
@@ -22,9 +25,12 @@ symbols:
 
 # Complex symbols
 . sentence ending	dot	all	always
+。 sentence ending	dot	all	always
```

Comment:
In English, should have a name indicating the charater name and allowing to differenciate it from English one.

```suggestion
。 sentence ending	ideographic period	all	always
```